scripts/generators/api_dump_generator.py

```diff
@@ -610,6 +610,8 @@ def generate_implementation(self, video=False):
 
             self.write('')
             for member in struct.members:
+                if (struct.name == 'StdVideoH265HrdParameters' and (member.name == 'pSubLayerHrdParametersNal' or member.name == 'pSubLayerHrdParametersVcl')):
```

Comment:
this seems very hacky, we have any idea why this happened?

Reply:
Yes, the length of these arrays comes from another struct, I opened an issue here #2482